recipes-kernel/linux/linux-iot2050-6.1.inc

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) Siemens AG, 2018-2022
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require recipes-kernel/linux/linux-custom.inc
+
+def get_patches(d, patchdir):
+    import glob
+
+    files_dir = os.path.join(d.getVar('THISDIR'), 'files')
+    patch_glob = os.path.join(files_dir, patchdir, '*.patch')
+    patches = glob.glob(patch_glob)
+    patches.sort()
+
+    return ' '.join(['file://' + patch[len(files_dir)+1:] for patch in patches])
+
+SRC_URI += " \
+    https://git.kernel.org/pub/scm/linux/kernel/git/cip/linux-cip.git/snapshot/linux-cip-${PV}.tar.gz \
```

Comment:
We have https://cdn.kernel.org/pub/linux/kernel/projects/cip/6.1/ now - requesting snapshots failed too often.